src/tr2/game/game_string.def

```diff
@@ -39,3 +39,5 @@ GS_DEFINE(OSD_LIGHTNING_CONTRAST_FMT, "Lighting Contrast: %s")
 GS_DEFINE(OSD_SCALER_FMT, "Scaler: x%d")
 GS_DEFINE(OSD_HARDWARE_RENDERING, "Hardware rendering")
 GS_DEFINE(OSD_SOFTWARE_RENDERING, "Software rendering")
+GS_DEFINE(OSD_PERSPECTIVE_FILTER_ON, "Perspective correction: on")
+GS_DEFINE(OSD_PERSPECTIVE_FILTER_OFF, "Perspective correction: off")
```

Comment:
Is there a reason it's labelled something different in TR2? I also didn't notice any different with it on vs off.

Reply:
Linter complains that it makes no sense to define this string in trx if it's only used in tr2 now.